docs/_static/vcr/vcr_v2.yaml

```diff
@@ -487,6 +493,18 @@ components:
           description: RFC3339 time string until when the credential is valid.
           type: string
           example: "2012-01-02T12:00:00Z"
+        credentialSubject:
+          $ref: '#/components/schemas/CredentialSubject'
+        withStatusList2021Revocation:
```

Comment:
maybe we should by default use statuslist2021 when the issuer is a did:web DID, instead of clients it having to enable it explicitly?

Side thought: we should never publish VCs which's subject is a did:web to the network? (issuer = did:web and subject=did:nuts could work in some weird use case)